doc/lspconfig.txt

```diff
@@ -169,10 +169,11 @@ passed overrides to `setup {}` are:
   Callback invoked by Nvim's built-in client when attaching a buffer to a
   language server. Often used to set Nvim (buffer or global) options or to
   override the Nvim client properties (`server_capabilities`) after a
-  language server attaches. Most commonly used for settings buffer
-  local keybindings. See |lspconfig-keybindings| for a usage example.
+  language server attaches.
 
-- {settings} `table <string, string|table|bool>`  
+  Prefer using an |LspAttach| autocommand handler instead.
```

Comment:
README mentions `on_attach` instead of LspAttach, might be worth changing it there.

Reply:
Done